### ISSUE TITLE
The digiInfo don't download.

### ISSUE THREAD

**Issue by kermitchang** [MicrosoftDocs/feedback#1404]
**🛑 IMPORTANT**: You can get your feedback addressed faster if you **use the comment section for the article in which you encountered a problem**.

**Link to article:**
Tell us where exactly a problem occurs.
In the link
https://docs.microsoft.com/zh-tw/windows-hardware/design/component-guidelines/simultaneous-pen-and-touch-validation
Under the page, the "digiinfo" of test tool can't download.

**Problem:**
What's wrong with the article?
In the link
https://docs.microsoft.com/zh-tw/windows-hardware/design/component-guidelines/simultaneous-pen-and-touch-validation
Under the page, the "digiinfo" of test tool can't download.

**Comment by EliotSeattle** [comment]
Thanks, @kermitchang.

That tool was available through our Connect portal, which has been retired. I have reached out to the Program Manager of the Pen and Ink hardware development and will update here as soo as I know anything about getting Digiinfo into your hands.

Eliot

**Comment by ejluttmann** [comment]
Any update on where this tool can be downloaded?

**Comment by EliotSeattle** [comment]
Hi All.

I apologize profusely for the delay. I had a string of issues uploading the files. But I persisted, and they can be downloaded at [https://download.microsoft.com/download/C/8/7/c8729e82-feca-482b-801d-f65979615003/digiinfo-19h1.zip](https://download.microsoft.com/download/C/8/7/c8729e82-feca-482b-801d-f65979615003/digiinfo-19h1.zip). I have updated the link in the en-us version of the referenced topic. Other locales will be picked up by our localization teams.

Thanks for your patience.

Eliot
